packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -156,6 +156,13 @@ const double _fontSizeToScale = 14.0;
 ///
 /// {@macro flutter.widgets.RawDialogRoute}
 ///
+/// {@tool dartpad}
+/// This sample demonstrates how to create a basic date picker.
+/// Tapping the button calls [showDatePicker] and displays the selected date.
```

Comment:
```suggestion
/// Tapping the button calls [showDatePicker] and displays the selected date.
```

It seems little confusing, tapping the button doesn't display selected date in the example


Maybe
`Tapping the button displays a date picker which returns the selected date.`